### DISCUSSION TITLE
email confirmation

### DISCUSSION THREAD

**Discussion by Mohammed-Shakeeb-ML** [community/community#7633] [Copilot Conversations] [answered]
I have signup into github copilot when will i get confirmation to access the application on vs code editor

**Comment by davecheney** [comment, marked as answer]
We’re adding people as we have capacity, thanks for your patience and understanding.

**Reply by Mohammed-Shakeeb-ML** [reply]
now current issue is mentioned below @davecheney, thanks for you response  
`GitHub Copilot could not connect to server. Extension activation failed: "getaddrinfo EAI_AGAIN api.github.com"`

**Reply by davecheney** [reply]
something on your network is preventing your computer from resolving api.github.com

**Comment by dilunkreatif2626** [comment]
I have registered for a github copilot account but the github copilot extension and confirmation email haven't arrived yet

**Reply by davecheney** [reply]
If you have not recieved and email then you are on the waitlist. We’re adding people as we have capacity, thanks for your patience and understanding.
